src/main/java/org/springframework/cloud/deployer/spi/scheduler/kubernetes/KubernetesScheduler.java

```diff
@@ -190,4 +210,12 @@ private void setImagePullSecret(ScheduleRequest scheduleRequest, CronJob cronJob
 					.add(localObjectReference);
 		}
 	}
+
+	private boolean isScheduleNameTooLong(String message ) {
```

Comment:
maybe `validationScheduleNameLength` or similar -- mainly to match something liek the other `validateScheduleName`